bin/serve.js

```diff
@@ -156,7 +156,8 @@ const startEndpoint = (endpoint, config) => {
 		if (typeof details === 'string') {
 			console.log(info(`Accepting connections at ${details}`));
 		} else if (typeof details === 'object' && details.port) {
-			console.log(info(`Accepting connections at http://${details.address}:${details.port}`));
+			const address = details.address === '::' ? 'localhost' : details.address;
```

Comment:
I don't think that `::` is the same as `localhost`. I believe `::1` is localhost, and `::` is similar to `0.0.0.0` in IPv4.